src/pages/Swap/index.tsx

```diff
@@ -328,14 +329,16 @@ export function Swap({
   const handleTypeInput = useCallback(
     (value: string) => {
       onUserInput(Field.INPUT, value)
+      maybeLogFirstSwapAction(trace)
```

Comment:
what's the total list of first swap actions? tapping input, output, max, & switch arrows? Could it be added to `src/tracing/SwapEventTimestampTracker.ts` for reference?